config/database.sql

```diff
@@ -57,7 +57,7 @@ CREATE TABLE `assignments`
 
 CREATE TABLE `course`
 (
-  `id`          int(11)                                        NOT NULL,
+  `id`          varchar(64)                                    NOT NULL,
```

Comment:
Primary key should be an int, for indexing, and the new "hash" should be called something else

Reply:
This new unique generated id, should be a new column